Makefile

```diff
@@ -149,7 +163,8 @@ CFLAGS :=                                   \
 	$(OPT) -std=c11 -MMD -MP                \
 
 ifneq ($(CFG_DIR),)
-CFLAGS += -include $(CFG_DIR)/board/$(TARGET_BOARD).h
+CFLAGS += -include $(BOARD_CONFIG_HEADER)
+CFLAGS += $(C_GENERAL_CONFIG_INCLUDES)
```

Comment:
Só comentando que desse jeito a gnt tem que de fato incluir no código os arquivos dentro da pasta cfg né? Não sei se a ideia era pra ser assim mesmo, mas é pq os de config de placa a gnt não inclui explicitamente, ai não sei direito o que é melhor. Acho que se for pra incluir manualmente, daria para deixar direto dentro de inc/, e não nesse diretório especial, mas acho que vale a discussão